dashboard/test/ui/features/learning_platform/script_overview.feature

```diff
@@ -22,8 +22,13 @@ Feature: Script overview page
     Then I verify progress for lesson 29 level 4 in detail view is "perfect"
     When I click selector ".teacher-panel table td:contains(Sally)" once I see it
     And I wait until element "td:contains(Maze)" is visible
+    # verify name format in summary view
+    And element "td:contains(2. Maze)" is visible
     Then I verify progress for lesson 2 level 1 is "perfect"
     Then I verify progress for lesson 2 level 2 is "not_tried"
+    And I reload the page
```

Comment:
Just curious, why is it important that we reload here?

Reply:
To make sure that the page displays the correct view (summary) on load. What was being tested before was if you switch from a teacher to a student (loads async on script overview) that summary view is displayed.